src/main/java/com/stripe/model/SetupIntentTypeSpecificPaymentMethodOptionsClient.java

```diff
@@ -0,0 +1,11 @@
+// File generated from our OpenAPI spec
+package com.stripe.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class SetupIntentTypeSpecificPaymentMethodOptionsClient extends StripeObject {}
```

Comment:
What is this class - do we need to add something in the changelog?

Reply:
It should not be generated. There's a bug in the spec.